lib/s3middleware/objectLegalHold.js

```diff
@@ -0,0 +1,111 @@
+const { parseString } = require('xml2js');
+const errors = require('../errors');
+
+/*
+    Format of the xml request:
+    <LegalHold>
+        <Status>ON|OFF</Status>
+    </LegalHold>
+*/
+
+/**
+ * validate status - validates status xml
+ * @param {object} status - parsed status object
+ * @return {boolean} - true or false on success, error on failure
+ */
+function _validateStatus(status) {
+    const validatedStatus = {};
+    const expectedValues = ['OFF', 'ON'];
```

Comment:
I would declare these once and reuse it, also Set is generally preferred for checking if an item exists in some collection as lookups are faster.